### PR TITLE
Split release table by channel

### PR DESCRIPTION
**Pull request by steverydz**
## Done
Implemented mobile designs for the release UI

## QA
- Go to https://snapcraft-io-3990.demos.haus/core/releases
- Resize to mobile size
- Check that it matches [the designs](https://app.zeplin.io/project/61b3231e21194c4a48005226/screen/61ddabca6985e510c8505f0a)

## Issue
Fixes https://github.com/canonical-web-and-design/marketplace-tribe/issues/2481, https://github.com/canonical-web-and-design/marketplace-tribe/issues/2483

### CODE REVIEW

**Comment by steverydz** [comment]
@anasereijo This is the final PR for the release UI mobile build so you can review the whole thing.

File: static/js/publisher/release/components/releasesTable/releaseCell.js [hunk 1]
```diff
@@ -92,6 +92,7 @@ const ReleasesTableReleaseCell = (props) => {
   };
 
   function handleHistoryIconClick(arch, risk, track, branchName) {
+    window.scrollTo(0, 0);
```
Comment (Lukewh) [inline static/js/publisher/release/components/releasesTable/releaseCell.js:95]: I always think when we do this we should store where the user came from (scroll) so when they close we can send them back - not for now, just to think about.

**Review by Lukewh — APPROVED** [review]
:+1: LGTM

**Comment by anasereijo** [comment]
**- [ ] some design issues**
how it looks like:


how it should look like:


**- [ ] can we have the filter by track dropdown?**
how it looks like:


how it should look like:


**- [ ] when clicking on a revision there are some design issues:** please check the [design](https://app.zeplin.io/project/61b3231e21194c4a48005226/screen/61e18c10f4e78e4fe0595ef9)



**- [ ] is it possible to match the design for the 'show more' revisions?**
how it looks like:


how it should look like:

**Comment by steverydz** [comment]
@anasereijo I've addressed all of your points. I've made issues for the track filter and the pagination because neither of those are trivial changes.

https://github.com/canonical-web-and-design/snapcraft.io/issues/4000
https://github.com/canonical-web-and-design/snapcraft.io/issues/4001

**Comment by anasereijo** [comment]
Thanks @steverydz

**Pull request merged**